.dep-versions

```diff
@@ -3,4 +3,5 @@ jax=0.4.23
 mhlo=4611968a5f6818e6bdfb82217b9e836e0400bba9
 llvm=cd9a641613eddf25d4b25eaa96b2c393d401d42c
 enzyme=1beb98b51442d50652eaa3ffb9574f4720d611f1
-pennylane=95129a0d6365b48cb4acfa828ceb6a8532e47ef5
+pennylane=cb6f1cdb8cda09eae9113284eb12b9dde18544a2
+lightning=efb946a736f824835ebf4fd692568d49e6cf1ca5
```

Comment:
Would you update the runtime to also use this commit from `.dep-versions` to build LQ & LK backend devices?